introduction/architecture.md

```diff
@@ -132,20 +132,19 @@ Back-end data nodes store data associated with distributed hypertables. Data nod
 are not aware of cluster-wide distribution policies, and otherwise look just as if they were
 single TimescaleDB instances. However, certain actions are blocked on data nodes to protect
 from accidentally executing operations on data nodes that might endanger the data consistency
-of distributed hypertables.
+of distributed hypertables. There are very few reasons to ever access a data node directly, and
+all accesses should be done at the access node level.
 
 It is important to note that access nodes and data nodes both run TimescaleDB, and for all intents and
-purposes, look just like a single instance of TimescaleDB from an operational perspective.
+purposes, look just like a single instance of TimescaleDB from an operational perspective. However, as
+mentioned above, all accesses should go through the access node, and you should not access the data
```

Comment:
I'd just clarify here "...all accesses for a distributed hypertable should go through..."

Reply:
Done!